Packs/CaseManagement-Generic/Scripts/CaseMgmtIncidentTypesDisplay/CaseMgmtIncidentTypesDisplay.py

```diff
@@ -0,0 +1,26 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+# Requirements - Create an XSOAR List called IncidentTypesFromList a list of comma separated Incident Types
+# Example: Incident Type 1,Incident Type 2, Incident Type 3
+
+# check if this is a new Incident or not
+incident = demisto.incident().get("id")
+
+# if new Incident, the ID will be empty:
+if not incident:
+    # get the XSOAR IncidentTypesFromList XSOAR List, and split on the comma
+    types_list = demisto.executeCommand("getList", {"listName": "IncidentTypesFromList"})[0]["Contents"].split(",")
```

Comment:
Please handle if getList fails.

Reply:
Sure.

Reply:
When you add error handling in case getList fails fails,
Do not forget **merge from master**

Thanks

Reply:
Done